specificatie/genereervariant/openapi.yaml

```diff
@@ -3094,9 +3058,9 @@ components:
     CrsEnum:
       type: string
       description: CRS van geometrie. epsg:28992 mapt op het RD New Nederlands coordinatenstelsel.
-      default: epsg:28992
       enum:
       - epsg:28992
+      default: epsg:28992
```

Comment:
waarom verplaats je de default nar achteren? Heeft dat voordelen?

Reply:
Dit betreft de gegenereerde variant, in de originele API specificatie is dit niet aangepast.